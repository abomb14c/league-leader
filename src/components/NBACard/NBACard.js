import React from 'react';

const NBACard = (props) => {
  const {name,
    market,
    wins,
    losses
  } = props;
  return (
    <div>
      <h3 className="team-name">{market} {name}</h3>
      <h3 className="record"> {wins}</h3>
      <h3>{losses}</h3>
    </div>
  );
};

export default NBACard;