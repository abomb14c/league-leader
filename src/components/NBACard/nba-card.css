.nba-card {
  align-items: center;
  border-top: 1px solid #C63D0F;
  background-color: #3B3738;
  display: flex;
  height: 42px;
  justify-content: space-between;
  padding-left: 2%;
  padding-right: 10%;
}

.english-desc {
  font-weight: 200;
  color: #FDF3E7;
}

.nba-title-container {
  text-align: left;
  width: 40%;
}

.english-rank {
  margin-left: 16%;
}

.english-record-container {
  display: flex;
  margin-left: 15%;
}

.english-points {
  margin-left: 15%;
}

.nba-team-name {
  color: #FDF3E7;
  font-weight: 200;
}

.nba-record {
  color: #FDF3E7;
  font-weight: 200;
}

.nba-record-container {
  width: 11%;
}