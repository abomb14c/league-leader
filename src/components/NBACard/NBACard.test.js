import React from 'react';
import { shallow } from 'enzyme';
import  NBACard from './NBACard';

describe('Leagues', () => {
  let wrapper;
  let mockProps;

  beforeEach(() => {

    mockProps = { 
      name: 'Bulls',
      market: 'Chicago',
      wins: 44,
      losses: 20
    };

    wrapper = shallow(<NBACard
      {...mockProps}
    />);
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});