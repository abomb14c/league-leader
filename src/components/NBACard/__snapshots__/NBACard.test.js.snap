// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Leagues should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NBACard
    losses={20}
    market="Chicago"
    name="Bulls"
    wins={44}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="nba-title-container"
>
          <h3
                    className="nba-team-name"
          >
                    Chicago
                     
                    Bulls
          </h3>
</div>,
        <div
          className="nba-record-container"
>
          <h3
                    className="nba-record"
          >
                     
                    44
                    -
                    20
          </h3>
</div>,
      ],
      "className": "nba-card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h3
            className="nba-team-name"
>
            Chicago
             
            Bulls
</h3>,
          "className": "nba-title-container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Chicago",
              " ",
              "Bulls",
            ],
            "className": "nba-team-name",
          },
          "ref": null,
          "rendered": Array [
            "Chicago",
            " ",
            "Bulls",
          ],
          "type": "h3",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h3
            className="nba-record"
>
             
            44
            -
            20
</h3>,
          "className": "nba-record-container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              " ",
              44,
              "-",
              20,
            ],
            "className": "nba-record",
          },
          "ref": null,
          "rendered": Array [
            " ",
            44,
            "-",
            20,
          ],
          "type": "h3",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="nba-title-container"
>
            <h3
                        className="nba-team-name"
            >
                        Chicago
                         
                        Bulls
            </h3>
</div>,
          <div
            className="nba-record-container"
>
            <h3
                        className="nba-record"
            >
                         
                        44
                        -
                        20
            </h3>
</div>,
        ],
        "className": "nba-card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h3
              className="nba-team-name"
>
              Chicago
               
              Bulls
</h3>,
            "className": "nba-title-container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Chicago",
                " ",
                "Bulls",
              ],
              "className": "nba-team-name",
            },
            "ref": null,
            "rendered": Array [
              "Chicago",
              " ",
              "Bulls",
            ],
            "type": "h3",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h3
              className="nba-record"
>
               
              44
              -
              20
</h3>,
            "className": "nba-record-container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " ",
                44,
                "-",
                20,
              ],
              "className": "nba-record",
            },
            "ref": null,
            "rendered": Array [
              " ",
              44,
              "-",
              20,
            ],
            "type": "h3",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
